azure.go

```diff
@@ -279,32 +332,29 @@ func (ac *AzureClient) listFromResourceGroup(resourceGroup string, resourceTypes
 		return nil, fmt.Errorf("Error: %v", err)
 	}
 	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+
 	if resp.StatusCode != 200 {
-		return nil, fmt.Errorf("Unable to query resource group API with status code: %d", resp.StatusCode)
+		return nil, fmt.Errorf("Unable to query resource API with status code: %d with body: %s", resp.StatusCode, body)
```

Comment:
Should be (I think):
```suggestion
		return nil, fmt.Errorf("Unable to query API with status code: %d and with body: %s", resp.StatusCode, body)
```